eBoard.md

```diff
@@ -12,22 +12,22 @@ published: true
+Hi! I'm Brady Cassada, the current Communications Coordinator of SSE. I am a Junior Software Engineering student at MSOE and I have been interning at Collins Aerospace since last summer. I have a passion for all different types of software and I love expanding my knowledge. In my free time I like playing hockey, making music, and playing video games. 
 
 ### Administrative Coordinator - Nicholas Scharrer
-Nicholas is a Junior Computer Engineering student. He's currently working as a Software Engineering Intern at Rockwell Collins. Outside of SSE he's a member of MSOE's NCAA DIII Men's Lacrosse Team.
+Hi, I'm Paul Rinaldi! I'm serving as the Administrative Coordinator as a Second-Year student for the 19-20 school year. This summer I'll be working at Collins Aerospace, but in my free time I'm also a software developer in two startup companies. Fun fact: I'm a quadruplet (2 of us at MSOE), do you know my brother? I'm eager to learn new technologies but I'm currently focusing on web development, redeveloping games for the old SNES, and participating in National Cyber League competitions. If you see me walking around, say hi!
 
 ### Events Coordinator - Naomi Yonekura
-Naomi is a senior software engineering student. Her main skill set falls under web development, but she's getting into machine learning and Unity. She's currently working as an intern at Rokkincat. Outside of SSE she enjoys playing games, baking, and listening to her favorite podcast The Adventure Zone. 
+Hey, my name is Errin Miller and I am the Event Coordinator for SSE. I grew up around the Milwaukee area, and as of the ’19-’20 school year, I am a sophomore computer science student with an interest in cybersecurity and data science. Outside of school, I spend most of my time reading, writing, and playing the piano. I look forward to working with you all this year!
 
 ### Programming Competition Team Capt. - Connor Hibbs
-Connor is a senior software engineering student. His favorite part of programming  is using it to solve specific problems and make life easier for people. He's currently working as an intern at Direct Supply and has previously interned at Enforcement Technology Group, Inc. working on police tatical systems. Outside of SSE he likes working on personal projects. His favorite personal projects reolve around IoT and automation, or developing multiplayer mobile party games.
+Hi, I’m Kiel Dowdle and I'm serving as the competitive programming team captain. I’m a sophomore nontraditional transfer student studying computer science. Outside of SSE I work as a carpenter and in my free time I enjoy playing sports, video games, and traveling. I'll be playing volleyball at Bradford beach over the summer and my favorite video game right now is Tetris 99.
 
 ### Webmaster - Grace Fleming
 Grace is a softmore software engineering student. She's currently working as a Software Engineering Intern at Direct Supply. Outside of SSE she plays piano, walks dogs, and computer networking.
```

Comment:
This is written in the third person while all the other ones are written in first person